src/vm/mod.rs

```diff
@@ -355,7 +359,7 @@ fn eval_all(
  */
 pub fn execute(program: &str) -> Result<Option<Value>> {
     let contract_id = QualifiedContractIdentifier::transient();
-    let mut contract_context = ContractContext::new(contract_id.clone());
+    let mut contract_context = ContractContext::new(contract_id.clone(), ClarityVersion::Clarity1);
```

Comment:
Should we use `info!` or some type of debug statement here (to alert a cli user to what clarity version is being used by default?)

Reply:
Yep, added in ab8204133